File: Selenium/Project/CRM_activity2.java
package activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeSuite;
import org.testng.annotations.Test;

public class CRM_activity2 {
        WebDriver driver = new FirefoxDriver();
        @BeforeSuite
        public void beforeClass() {
                System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
                WebDriverManager.firefoxdriver().setup();
                driver.get("http://alchemy.hguy.co/crm");
        }

        @Test
        public void headerImageURL(){
                String pgeTitle = driver.getTitle();
                System.out.println("The Page Title is: "+pgeTitle);
//                WebElement logo = driver.findElement(By.tagName("src"));
                WebElement logo1 = driver.findElement(By.xpath("//img[@alt='SuiteCRM']"));

                String strLogo = logo1.getAttribute("src");

                System.out.println("The URL of the Header Image is: "+strLogo);
                System.out.println("Run Completed");
        }
        @AfterClass
        public void afterClass(){
                driver.quit();
        }
}

File: Selenium/Project/CRM_activity2.py
from selenium.webdriver.common.by import By
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions as EC

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    # initialize the wait object
    wait = WebDriverWait(driver, 10)

    # Navigate to the URL
    driver.get("http://alchemy.hguy.co/crm")
    # Print the title of the page
    print("Page title is: ", driver.title)

    logo1 = driver.find_element(By.XPATH, "//img[@alt='SuiteCRM']")
    strLogo = logo1.get_attribute("src")
    print("The URL of the Header Image is: "+strLogo)
    # assert driver.title == "SuiteCRM"
    print("Run Completed")

    # close the browser
    driver.quit()
